app/views/videos/show/_translate_frame.html.erb

```diff
@@ -0,0 +1,47 @@
+<% if translation %>
+  <div id="translate">
+    <div class="panel-tabs">
+      <ul class="nav nav-tabs" role="tablist">
+        <li role="presentation" class="active tab-translation">
+          <a href="#tab-amara" class="panel-title panel-title-tabs" aria-controls="home" role="tab" data-toggle="tab">
+            Translate on Amara
+          </a>
+        </li>
+        <li role="presentation" class="tab-translation">
+          <a href="#tab-srt" class="panel-title panel-title-tabs" aria-controls="srt" role="tab" data-toggle="tab">
+            Translate Offline
+          </a>
+        </li>
+      </ul>
+    </div>
+
+    <div class="panel panel-default module module-padding module-tabs">
+      <div role="tabpanel">
+        <div class="tab-content">
+          <div role="tabpanel" class="tab-pane fade in active tab-amara" id="tab-amara">
+            <span>If you'd like to use Amara's translation interface, you can use the <b><a href="#">link to the Amara video</a></b>. Please, just make sure to let us know when you're done.</span>
```

Comment:
The "Please, just make sure to let us know when you're done" is redundant since there is a field that tells them that they can earn points by submitting their completed Amara link.